## RNA-Seq workflow tutotial

http://www.bioconductor.org/help/workflows/rnaseqGene/

RNA-seq workflow: gene-level exploratory analysis and differential expression

Michael Love [1], Simon Anders [2,3], Vladislav Kim [3], Wolfgang Huber [3]
[1] Department of Biostatistics, Dana-Farber Cancer Institute and Harvard School of Public Health, Boston, US;
[2] Institute for Molecular Medicine Finland (FIMM), Helsinki, Finland;
[3] European Molecular Biology Laboratory (EMBL), Heidelberg, Germany.
Short links

`-Counting reads`

`-Building a DESeqDataSet`

`-Exploratory analysis and visualization`

`-Differential expression`

`-Plotting results`

`-Annotating and exporting results`

`-Accounting for unknown batches`

`-Time course experiments`

`-Session information and references`

Abstract

Here we walk through an end-to-end gene-level RNA-seq differential expression workflow using Bioconductor packages. We will start from the FASTQ files, show how these were aligned to the reference genome, and prepare a count matrix which tallies the number of RNA-seq reads/fragments within each gene for each sample. We will perform exploratory data analysis (EDA) for quality assessment and to explore the relationship between samples, perform differential gene expression analysis, and visually explore the results.
